src/index.js

```diff
@@ -92,6 +97,11 @@ export async function run() {
   }
 }
 
+function didTmateQuit() {
+  const tmateSocketPath = process.platform !== "win32" ? "/tmp/tmate.sock" : "C:/msys64/tmp/tmate.sock"
```

Comment:
```suggestion
  const tmateSocketPath = process.platform === "win32" ? "C:/msys64/tmp/tmate.sock" : "/tmp/tmate.sock"
```
nit: I personally think its cleaner to check first for the === case since then the if and its related win32 condition is next to each other.

Reply:
I just imitated what is done in the function below :-P

Will change.

Reply:
Pushed.

Reply:
yeah, adjusted it in the [other PR](https://github.com/mxschmitt/action-tmate/pull/47/files#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556L96).

Reply:
About that... I finally found a better way (because I finally figured out how, exactly, `CHERE_INVOKING` was not set).